controllers/storagecluster/odfinfoconfig.go

```diff
@@ -137,6 +137,12 @@ func getOdfInfoData(r *StorageClusterReconciler, storageCluster *ocsv1.StorageCl
 		return "", err
 	}
 
+	annotation := map[string]string{}
+	ocsApiEndpointExportedAddress := storageCluster.GetAnnotations()[ocsv1a1.OcsApiEndpointExportedAddress]
+	if ocsApiEndpointExportedAddress != "" {
+		annotation[ocsv1a1.OcsApiEndpointExportedAddress] = ocsApiEndpointExportedAddress
+	}
```

Comment:
Please copy all annotations from the storage cluster under the OCS domain. We should not have special code to just handle this annotation. That was the main point behind choosing an exporting mechanism